### ISSUE TITLE
Running test recipes in README

### ISSUE THREAD

**Issue by alexmojaki** [oughtinc/ice#7]
The README suggests in https://github.com/oughtinc/ice#running-tests:

> Cheap integration tests:
> 
> ```
> ./scripts/run-recipe.sh --mode test
> ```

This had me confused for a while.

1. This prompts me select a recipe, which is already surprising. I expected it to automatically pick parameters for testing, I don't know which recipe I should select.
2. The first few choices in the list all require papers, so selecting any of them gives a long traceback ending in `TypeError: missing a required argument: 'paper'` which *looks* more like a fundamental bug than incorrect user inputs.
3. Once I figured out that I needed to specify other arguments to pass papers, it wasn't clear what those arguments should be. The CLI help doesn't offer any example values. Maybe the README could suggest something like:

```
./scripts/run-recipe.sh --mode test --recipe-name AdherenceKeywordBaseline --input-files ./papers/keenan-2018-tiny.txt
```

4. Running with those arguments gives output ending abruptly in a title saying `Final result for keenan-2018-tiny.txt` with nothing underneath. Where are these results of which it speaks? Did the test pass? Should I use different arguments?
5. Running the unit tests with `./scripts/run-tests.sh` runs all recipes with and without papers automatically.
    1. Why do the 'unit tests' apparently include 'integration tests'?
    2. Why suggest `run-recipe.sh` if `run-tests.sh` takes care of this already?

**Comment by lslunis** [comment]
I agree that this is pretty confusing. Most of the scripts have pretty sharp edges and I'd like to revise some of them and hopefully get rid of others. I'll determine priority next week once the upcoming ICE release is done.
